ironfish/src/blockchain/blockchain.ts

```diff
@@ -1520,3 +1520,17 @@ export class VerifyError extends Error {
     this.score = score
   }
 }
+
+export enum BlockCreationErrorReason {
+  HEAD_CHANGED = 'HEAD_CHANGED',
+}
```

Comment:
Since there's only one special case here right now, could we just have a `HeadChangedError`?

Reply:
Yea fair